Added App Proxy Extension Integration

japere opened the pull request:
Added a bullet point on the app proxy integration in the extension. Go live date 5/4/18.

azurecla commented:
Hi __@masnider__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (masnider). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Ja-Dunn commented:
@masnider - Hi, thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

ktoliver commented:
@masnider - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-communication). Thanks for your understanding.

japere committed:
Added App Proxy Extension Integration

Added a bullet point on the app proxy integration in the extension. Go live date 5/4/18.
